go/oasis-test-runner/oasis/network.go

```diff
@@ -641,6 +641,8 @@ func (net *Network) startOasisNode(
 		cfg.Consensus.UpgradeStopDelay = 10 * time.Second
 
 		extraArgs = extraArgs.debugAllowDebugEnclaves()
+	} else {
+		baseArgs = append(baseArgs, "--genesis.file", net.GenesisPath())
```

Comment:
Not sure why we need this and why only in `else` statement. We already defined the file with `cfg.Genesis.File`.

Reply:
It is for some non-node debug subcommands that do their own things and should actually be moved into separate binaries or removed.